#  âœ‚ï¸ðŸ·ï¸ Cut&TAG_MSci
Repository and Resources for MSci data science project in the Soutoglou Laboratory

This repository includes a mixture of worsheets, information pages, and links to various other learning resources. 
You can get the most out of the materials by following the order of the resources (L1, L2, etc.) and compeleting the worksheets in Google Collab. 

---

### Here is a brief repository indez: 

> #### L1_background.md
> This information page gives you an overview and comparison points for the CUT&Run methodology that this project focuses on.

> #### L2_filetypes.md
> Introduction to the various file types you will encounter with CLI scrippting in NGS bioinformatics workflows. 
Critically, first introduction to the workflow we will be using in this project.

> #### L3_shellbasics.md and L3_bash_worksheet.ipynd
> Includes cheatsheets for useful shell commands, links to other similar resources, and importantly the set up instructions for MacOS
> Worksheet goes over basic and then more intermediate bash. Important to familiarise yourself with this.

---

#### Forkflow sorksheets
The following worksheets can be used as a primer for and should give you some practice with using some of the most commonly used command line tools for NGS analysis 

> #### L4_bowtie2_samtools_worksheet.ipynb
> Use a toy genome and reads to get to grips with Bowtie2 and samtools

> #### L5&6_MAPQ_filtering.ipynb
> Important basis for QC:
> Using a toy genome and reads, practice Bowtie2 and samtools application and learn how to inspect outputs for mapping quality. 

